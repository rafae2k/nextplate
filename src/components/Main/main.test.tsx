import { render, screen } from '@testing-library/react'

import { Main } from '.'

describe('<Main/>', () => {
  it('should render the heading', () => {
    const { container } = render(<Main>Hello World</Main>)

    const heading = screen.getByRole('heading', { name: /Hello World/i })

    expect(heading).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
