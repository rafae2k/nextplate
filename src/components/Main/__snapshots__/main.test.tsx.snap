// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render the heading 1`] = `
<h1>
  Hello World
</h1>
`;
